main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/btcsuite/btcd/btcec"
+)
+
+var ()
+
+func main() {
+	conf_flag := "clightning"
```

Comment:
This should be fetched from env variable (RUN_CLIGHTNING) and better have another env variable also for lnd (RUN_LND) because eventually they will run simultaniousely.